### DISCUSSION TITLE
How to add a column to table/update existing db schema

### DISCUSSION THREAD

**Discussion by Panda-Sharp** [linq2db/linq2db#3212] [Q&A] [answered]
I have an existing SQLite DB in a UWP app, I want to add a column to a table, what's the best way to assure that the column will be added when the user will update (start) the new version of the app? thanks

**Comment by MaceWindu** [comment, marked as answer]
linq2db doesn't provide any migration APIs right now, so your option with linq2db it to use raw sql

**Comment by sdanyliv** [comment]
Usually [FluentMigrator](https://github.com/fluentmigrator/fluentmigrator) is an option for defining migrations. linq2db itself do not change tables.
